### PR TITLE
chore: add dynamic settings table to README

### PR DESCRIPTION
**Pull request by alexey-ban**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,15 @@ Dynamic settings include:
 * Access Permissions
 * Rate Limits
 
+|Parameter|Description|
+|---------|-----------|
+|routes|Path(s) to route to specific upstreams or to respond with configured body.|
+|route-rate|Parameters for vote endpoint.|
```
Comment (astsiapanay) [inline README.md]: this parameter is obsolete. Please remove it.

File: README.md [hunk 2]
```diff
@@ -90,6 +90,15 @@ Dynamic settings include:
...
+|applications|A list of deployed AI DIAL Applications and their parameters:<br />`endpoint`: AI DIAL Application API for chat completions.<br />`iconUrl`: a path to the icon used for the AI DIAL Application on the UI.<br />`description`: a brief description of the AI DIAL Application rendered on the UI.<br />`displayName`: a name of the AI DIAL Application used on the UI.|
+|models|A list of deployed models and their parameters:<br />`type`: specify `chat` or `embedding` model type.<br />`iconUrl`: a path to the icon used for the model on the UI.<br />`description`: a brief description of the model rendered on the UI.<br />`displayName`: a name of the model rendered on the UI.<br />`endpoint`: model API for chat completions or embeddings.<br />`upstreams`: upstreams are used for load-balancing. A request will be sent to the configured model endpoint and will contain X-UPSTREAM-ENDPOINT and X-UPSTREAM-KEY headers:<br />`endpoint`: model endpoint.<br />`key`: your API key.|
```
Comment (astsiapanay) [inline README.md]: `key` is obsolete. Please remove

File: README.md [hunk 3]
```diff
@@ -90,6 +90,15 @@ Dynamic settings include:
...
+|keys|API Keys parameters:<br />`<proxyKey>`: your API key.<br />`project`: a project name this key is assigned to.<br />`role`: name of one of the configured roles. Defines permissions for the key.<br />`userAuth`: can be disabled/enabled/optional.<br />**Disabled** - Authorization header is ignored and not sent to upstream.<br />**Enabled** - Authorization header is required and sent to upstream. Optional - Authorization header is optional and sent to upstream if present.|
```
Comment (astsiapanay) [inline README.md]: `userAuth` is obsolete. Please remove

File: README.md [hunk 4]
```diff
@@ -90,6 +90,15 @@ Dynamic settings include:
...
+|roles|A list of configured roles with their limitations. Specify a role name and in limits, specify models (can be models, Applications, Addons, Assistants) and limitation configurations:<br />`minute`: the total tokens per minute limit that can be sent to the model is managed using a floating window approach. This technique ensures a well-distributed rate-limiting mechanism, allowing control over the number of tokens sent to the model within a defined time frame, typically a one-minute window.<br />`day`: the total tokens per day limit that can be sent to the model is managed using a floating window approach. This method ensures a balanced rate-limiting mechanism, allowing control over the number of tokens sent to the model within a specified time frame, typically a 24-hour window.<br />|
```
Comment (astsiapanay) [inline README.md]: More precise definition of the roles is a list of API key roles with their limits. So each API key has exactly one role and this role is defined in the list of roles here. A role is associated with a list of models, applications, assistants where limits are defined.

File: README.md (hunk 2 above)
Comment (alexey-ban) [reply README.md]: But how do we pass the Azure OpenAI key now?

**Review by astsiapanay — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (alexey-ban) [reply README.md]: Adjusted table style based on problem review

**Pull request merged**